Quests/QuestNPC.cs

```diff
@@ -14,18 +15,18 @@ public class QuestNPC : GlobalNPC
 
         private BaseQuest availableQuest;
 
-        private int questCheckTick = 0;
+        private int questCheckTick = 300;
 
         private BaseQuest GetCurrentAvailableQuest(NPC npc)
         {
             if (availableQuest != null) return availableQuest;
             if (questCheckTick > 0)
             {
-                questCheckTick++;
-                if (questCheckTick >= 600) questCheckTick = 0;
+                questCheckTick--;
+                if (questCheckTick <= 0) questCheckTick = 0;
                 return null;
             }
-            questCheckTick++;
+            //questCheckTick++;
```

Comment:
Maybe reset questCheckTick on this line? The way it works now, once questCheckTick reaches 0, it will start checkin quests every tick, which might be a bit much once there are more quests.